modules/i3s/src/lib/utils/customize-сolors.ts

```diff
@@ -16,20 +31,26 @@ import {I3STileAttributes} from '../parsers/parse-i3s-attribute';
  * @param options - loader options
  * @returns midified colors attribute
  */
+// eslint-disable-next-line max-params
 export async function customizeColors(
   colors: MeshAttribute,
-  featureIds: MeshAttribute,
-  tileOptions: I3STileOptions,
-  tilesetOptions: I3STilesetOptions,
+  featureIds: TypedArray,
+  attributeUrls: string[],
+  fields: Field[],
+  attributeStorageInfo: AttributeStorageInfo[],
+  colorsByAttribute: ColorsByAttribute | null,
   options?: I3SLoaderOptions
```

Comment:
Do we still need options? The function is not loader-related anymore

Reply:
We need token only. Updated in the latest commit